src/Adapter/ContainerBuilder.php

```diff
@@ -197,7 +203,7 @@ private function loadDoctrineAnnotationMetadata()
     /**
      * @param SfContainerBuilder $container
      *
-     * @throws \Exception
+     * @throws Exception
      */
     private function loadServicesFromConfig(SfContainerBuilder $container)
```

Comment:
Shouldn't all references to `SfContainerBuilder` be  changed as well?

Reply:
Yes, you're right